frontends/web/src/routes/bitsurance/bitsurance.tsx

```diff
@@ -71,51 +74,59 @@ export const Bitsurance = ({ accounts }: TProps) => {
         <GuidedContent>
           <Header title={<h2>{t('sidebar.insurance')}</h2>} />
           <View fullscreen={false}>
-            <ViewContent fullWidth>
+            <ViewContent>
               <p className={style.noVspace}>{t('bitsurance.intro.text1', { amount })}</p>
-              <p className={style.noVspace}>{t('bitsurance.intro.text2')} <A href="https://www.bitsurance.eu/">{t('bitsurance.intro.link')}</A>.</p>
-              <Grid col="2" textAlign="left">
-                <Column asCard>
-                  <h3 className="title">
-                    {t('bitsurance.insure.title')}
-                  </h3>
-                  <p>{t('bitsurance.insure.text')}</p>
-                  <ul className={style.clean}>
-                    <li><Checked/><span>{t('bitsurance.insure.listItem1')}</span></li>
-                    <li><Checked/><span>{t('bitsurance.insure.listItem2')}</span></li>
-                  </ul>
-                  <Button onClick={ maybeProceed } primary>
-                    {t('bitsurance.insure.button')}
-                  </Button>
-                  <A href="https://www.bitsurance.eu/faq/">{t('bitsurance.insure.faq')}</A>
-                </Column>
-                <Column asCard>
-                  <h3 className="title">
-                    {t('bitsurance.detect.title')}
-                  </h3>
-                  <p>{t('bitsurance.detect.text')}</p>
-                  {insuredAccounts.length > 0 ? (
+              <div className={style.gridContainer}>
+                <Grid col="2" textAlign="left">
+                  <Column asCard>
+                    <h3 className={style.title}>
+                      {t('bitsurance.insure.title')}
+                    </h3>
+                    <p className={style.cardBody}>{t('bitsurance.insure.text')}</p>
+                    <ul className={style.clean}>
+                      <li><Checked/><span>{t('bitsurance.insure.listItem1')}</span></li>
+                      <li><Checked/><span>{t('bitsurance.insure.listItem2')}</span></li>
+                      <li><Checked/><span>{t('bitsurance.insure.listItem3')}</span></li>
+                    </ul>
+                    <p className={style.cardBody2}>
+                      {t('bitsurance.insure.text2')} {' '}
+                      <A href="https://www.bitsurance.eu/">{t('bitsurance.intro.link')}</A>.
+                    </p>
+                    <Button onClick={maybeProceed} primary className={style.ctaButton}>
+                      {t('bitsurance.insure.button')}
+                    </Button>
```

Comment:
small nit: could you try if the buttons can just be used in `<ColumnButtons>` component?